test/cloudtest/pkg/utils/shell_utils_test.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/onsi/gomega"
+)
+
+func TestProcessOutputShouldNotLostOutput(t *testing.T) {
+	assert := gomega.NewWithT(t)
+	const expected = "output..."
+	for i := 0; i < 1000; i++ {
```

Comment:
How long it take to execute this kind of test?

Reply:
~3s
`<testcase classname="github.com/networkservicemesh/networkservicemesh/test/cloudtest/pkg/utils" name="TestProcessOutputShouldNotLostOutput" time="2.630000"/>` from JunitResult

Reply:
I think it is not required to eat almost 3 seconds for such not important case.

Reply:
@haiodo What the limit time should be per unit test?

Reply:
finally reduced time to 1s